test/powershell/Modules/Microsoft.PowerShell.Core/Get-Module.Tests.ps1

```diff
@@ -114,12 +143,38 @@ Describe "Get-Module -ListAvailable" -Tags "CI" {
         $modules[4].Path | Should -BeExactly (Resolve-Path "$testdrive\Modules\Zoo\Too\Zoo.psm1").Path
     }
 
-    It "Get-Module -FullyQualifiedName <FullyQualifiedName> -ListAvailable" {
-        $moduleSpecification  = @{ModuleName = "Foo"; ModuleVersion = "2.0"}
+    It "Get-Module -FullyQualifiedName @{ModuleName = '<Name>' ; ModuleVersion = '<ModuleVersion>'} -ListAvailable - <TestCaseName>" -TestCases $listModuleNameTestCases {
+        param(
+            [Parameter(Mandatory = $true)]
+            $Name,
+            $TestCaseName,
+            [Parameter(Mandatory = $true)]
+            $ExpectedName,
+            $ModuleVersion
+        )
+
+        $moduleSpecification  = @{ModuleName = $name ; ModuleVersion = $ModuleVersion}
         $modules = Get-Module -FullyQualifiedName $moduleSpecification -ListAvailable
         $modules | Should -HaveCount 1
-        $modules.Name | Should -BeExactly "Foo"
-        $modules.Version | Should -BeExactly "2.0"
+        $modules.Name | Should -BeExactly $ExpectedName
+        $modules.Version | Should -BeExactly $ModuleVersion
+    }
+
+    It "Get-Module -FullyQualifiedName @{ModuleName = '<Name>' ; ModuleVersion = '<ModuleVersion>'} - <TestCaseName>" -TestCases $loadedModuleNameTestCases {
```

Comment:
Why do we need the duplicate test? We could add the `$loadedModuleNameTestCases` in previous test.

Reply:
We need to test the other code path for loaded modules

Reply:
In the case I'd expect that test names say this explicitly and more clear. Otherwise I don't understand why we duplicate the code.

Reply:
Please provide a suggestion on how to say it more clearly that stating the command line that is being run.

Reply:
The difference in both test case names is that one has `-ListAvailable` the other does not. I think that makes it clear which code path is being tested.